contracts/mainnet/connectors/morpho-compound/main.sol

```diff
@@ -382,33 +378,6 @@ abstract contract MorphoCompound is Helpers, Events {
 			_setId
 		);
 	}
-
-	/**
-	 * @dev Claim rewards.
-	 * @notice Claim rewards for the given assets from underlying protocol.
-	 * @param _poolTokenAddresses The cToken addresses to claim rewards from..(For ETH: cToken address of WETH)
-	 * @param _tradeForMorphoToken Whether or not to trade COMP tokens for MORPHO tokens.
-	 */
-	function claim(
-		address[] calldata _poolTokenAddresses,
-		bool _tradeForMorphoToken
-	)
-		external
-		payable
-		returns (string memory _eventName, bytes memory _eventParam)
-	{
-		uint256 _amountOfRewards = MORPHO_COMPOUND.claimRewards(
-			_poolTokenAddresses,
-			_tradeForMorphoToken
-		);
-
-		_eventName = "LogClaimed(address[],bool,uint256)";
-		_eventParam = abi.encode(
-			_poolTokenAddresses,
-			_tradeForMorphoToken,
-			_amountOfRewards
-		);
-	}
```

Comment:
Hi @MerlinEgalite 
We will be creating a common rewards connector. Will be adding all the necessary claim functions to it. It will make the development easy in case there is any change in the future on rewards contracts.

Reply:
Perfect thx! Wanted to be sure you won't lose rewards